lua/entities/gmod_wire_egp/lib/egplib/usefulfunctions.lua

```diff
@@ -81,8 +80,8 @@ end
 -- Draw from top left
 --------------------------------------------------------
 
-function EGP:MoveTopLeft(ent, obj)
-	if not self:ValidEGP(ent) then return end
+function EGP.MoveTopLeft(ent, obj)
+	if not EGP:ValidEGP(ent) then return end
```

Comment:
Why are some 'EGP' using colon and some using period for function calls?

Reply:
Because there is only 2 calls of EGP.MoveTopLeft in the codebase. I see no benefit to them being methods instead of functions. It just passes the EGP table for no reason.

Reply:
is EGP just a namespace (i.e. table of functions)? Why is ValidEGP a method? You just didn't want to change them all yet?

Reply:
You're entirely correct, I just haven't gotten around to them yet. My "half-rule" was to try not to touch anything that's completely unrelated. Changing that function's signature would require changing it all over EGP.

Reply:
All right